### PR TITLE
Packaging v2

### PR DESCRIPTION
**Pull request by OniriCorpe**
https://github.com/YunoHost/doc/blob/resources-doc/pages/06.contribute/10.packaging_apps/14.advanced/03.packaging_v2/packaging_v2.md

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

### CODE REVIEW

**Comment by lapineige** [comment]
Do you need help on this ? :)

**Comment by lapineige** [comment]
:up: since Yunohost is now updated to support them by default ? :)

**Comment by OniriCorpe** [comment]
Hi @lapineige! 

> Do you need help on this ? :)

Feel free to contribute, I'm kinda tired so help is welcome!
(I'll stop here for tonight)

**Comment by OniriCorpe** [comment]
> I'm unsure of what's left to do 🤔

Yeah, me too :"3

maybe it's time to test?
and watch it fail miserably? \:D/

!testme

**Comment by OniriCorpe** [comment]
!testme

**Comment by lapineige** [comment]
https://ci-apps-dev.yunohost.org/ci/job/5568

**Comment by alexAubin** [comment]
>WARNING - ./install: line 226: password: unbound variable

Ah this sounds like an issue in YunoHost's core :sweat_smile:

**Comment by alexAubin** [comment]
(Should be fixed in https://github.com/YunoHost/yunohost/commit/9bd4344f25b079b66a329d93c1cd552e340cf9f9 to be released in next iteration)

**Comment by ericgaspar** [comment]
!testme

**Comment by OniriCorpe** [comment]
!testme

**Comment by OniriCorpe** [comment]
I'll resume work on this in the next few days :)

File: manifest.toml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+website = "https://docs.gotosocial.org/" 
+
+[integration]
+architectures = [
+  "amd64",
+  "i386",
+  "arm64",
+  "armv6",
+  "armv7",
+]
```
Comment (alexAubin) [inline manifest.toml:28]:
```suggestion
architectures = [ "amd64", "i386", "arm64", "armhf" ]
```

**Pull request closed without merging**